README.md
MonotopProdAODConfiguration
===========================

Configuration to produce AOD-liked samples from LHE to RECO


FastSim/history.txt
    6  cmsDriver.py Configuration/Generator/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
    7  cmsDriver.py ../../../../../../../../../../../../../../../../../../../../../opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
    8  cmsDriver.py   --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
    9  cmsDriver.py Configuration/../Configuration/Generator/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   10  cmsDriver.py Configuration/Generator/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   11  cmsDriver.py H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   12  cmsDriver.py --help
   13  cmsDriver.py --filein file:/opt/sbg/data/data4/cms/mbuttign/THESE/Monotop/Analysis_8TeV/CMSSW_5_3_11/src/MCProd_8TeV/test_config_TAU.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   14  cmsRun --filein_GEN_VALIDATION.py 
   15  cmsDriver.py file:/opt/sbg/data/data4/cms/mbuttign/THESE/Monotop/Analysis_8TeV/CMSSW_5_3_11/src/MCProd_8TeV/test_config_TAU.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   16  cmsDriver.py ../Generator/  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   17  cmsDriver.py ../Generator/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   18  cp /opt/sbg/data/data4/cms/mbuttign/THESE/Monotop/Analysis_8TeV/CMSSW_5_3_11/src/MCProd_8TeV/test_config_TAU.py /opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/ 
   19  ls 
   20  rm --filein_GEN_VALIDATION.py H200ZZ4L_cfi_py_GEN_VALIDATION.py
   21  rm H200ZZ4L_cfi_py_GEN_VALIDATION.py
   22  ls 
   23  rm "--filein_GEN_VALIDATION.py" 
   24  ls
   25  rm '--filein_GEN_VALIDATION.py' 
   26  rm ./--filein_GEN_VALIDATION.py
   27  ls
   28  cd src/MCProd_8TeV/
   29  ls 
   30  cmsDriver.py Generator/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   31  cmsDriver.py H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   32  cmsDriver.py ../H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   33  cmsDriver.py ~/opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   34  cmsDriver.py /opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   35  ls /opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/
   36  ls /opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/H200ZZ4L_cfi.py
   37  ls /opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/H200ZZ4L_cfi.py
   38  ll /opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/H200ZZ4L_cfi.py
   39  ls 
   40  ll config_prod.py
   41  ll /opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/
   42  ll /opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator
   43  cmsDriver.py ../python/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   44  cmsDriver.py test_config_TAU.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   45  cmsDriver.py test_confi  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   46  cmsDriver.py ../../../../../../../../../../../../../../../../../../../../../../../../../../opt/sbg/data/data4/cms/mbuttign/THESE/Monotop/Analysis_8TeV/CMSSW_5_3_11/src/MCProd_8TeV/test_config_TAU.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   47  cmsDriver.py --help
   48  man cmsDriver.py
   49  cmsDriver.py --help
   50  cmsenv
   51  top
   52  *
   53  top 
   54  l
   55  ls
   56  ll
   57  top 
   58  cmsenv
   59  cmsDriver.py test_confi  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   60  cmsenv
   61  cmsDriver.py test_confi  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   62  cmsDriver.py   --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   63  cmsDriver.py file:mbuttign@sbgui8:/opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/H200ZZ4L_cfi.py  --conditions auto:mc -s GEN,VALIDATION:genvalid_w --datatier GEN --eventcontent RAWSIM -n 10 --no_exec
   64  cd /opt/exp_soft/cms/slc5_amd64_gcc462/cms/cmssw/CMSSW_5_3_11/python/Configuration/Generator/
   65  ls
   66  grep -nr "Pythia8GeneratorFilter" ./
   67  gedit QCD_Pt_30_8TeV_pythia8_cff.py Pythia8_PhotonJetpt15_20_10TeV_8TeV_cff.py ADDMonoJet_8TeV_d3MD3_cfi.py MinBias_8TeV_pythia8_cff.py &

Prod_LHE/py.py

# py.py
# This file is automatically generated. Do not edit.
_tabversion = '3.2'

_lr_method = 'LALR'

_lr_signature = '\xf5\\}<\xf1\x11g\x0f}\xe5cl\xe3\xc4\x97\xac'
    
_lr_action_items = {'NUMBER':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,]),'COMPLEX':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,]),'ASEC':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,]),'CONJ':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,]),'POWER':([1,5,12,17,19,21,22,23,24,25,26,27,28,29,31,32,33,40,41,42,43,44,45,46,47,48,49,50,52,54,55,56,57,59,60,62,63,],[-11,-4,-5,-7,34,-15,-19,-7,34,34,34,-16,-13,-14,-12,-17,-18,34,-8,-9,-3,34,-6,34,34,34,34,34,-20,34,34,-10,-21,34,-27,34,-28,]),')':([1,5,12,17,21,22,25,26,27,28,29,31,32,33,41,42,43,44,45,46,47,48,49,50,52,54,55,56,57,59,60,62,63,],[-11,-4,-5,-7,-15,-19,43,-2,-16,-13,-14,-12,-17,-18,-8,-9,-3,52,-6,-23,-25,-24,-26,-22,-20,56,57,-10,-21,60,-27,63,-28,]),'(':([0,2,3,4,6,7,8,9,10,13,14,15,16,18,20,30,34,35,36,37,38,39,51,53,58,61,],[7,20,7,7,7,7,7,7,7,7,30,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,]),'+':([1,5,12,17,19,21,22,23,24,25,26,27,28,29,31,32,33,40,41,42,43,44,45,46,47,48,49,50,52,54,55,56,57,59,60,62,63,],[-11,-4,-5,-7,35,-15,-19,-7,35,35,-2,-16,-13,-14,-12,-17,-18,35,-8,-9,-3,35,-6,-23,-25,-24,-26,35,-20,35,35,-10,-21,35,-27,35,-28,]),'*':([1,5,12,17,19,21,22,23,24,25,26,27,28,29,31,32,33,40,41,42,43,44,45,46,47,48,49,50,52,54,55,56,57,59,60,62,63,],[-11,-4,-5,-7,36,-15,-19,-7,36,36,-2,-16,-13,-14,-12,-17,-18,36,-8,-9,-3,36,-6,36,-25,36,-26,36,-20,36,36,-10,-21,36,-27,36,-28,]),'-':([0,1,5,6,7,8,12,17,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,],[8,-11,-4,8,8,8,-5,-7,37,8,-15,-19,-7,37,37,-2,-16,-13,-14,8,-12,-17,-18,8,8,8,8,8,8,37,-8,-9,-3,37,-6,-23,-25,-24,-26,37,8,-20,8,37,37,-10,-21,8,37,-27,8,37,-28,]),',':([1,5,12,17,21,22,26,27,28,29,31,32,33,40,41,42,43,44,45,46,47,48,49,50,52,55,56,57,59,60,63,],[-11,-4,-5,-7,-15,-19,-2,-16,-13,-14,-12,-17,-18,51,-8,-9,-3,53,-6,-23,-25,-24,-26,-22,-20,58,-10,-21,61,-27,-28,]),'/':([1,5,12,17,19,21,22,23,24,25,26,27,28,29,31,32,33,40,41,42,43,44,45,46,47,48,49,50,52,54,55,56,57,59,60,62,63,],[-11,-4,-5,-7,38,-15,-19,-7,38,38,-2,-16,-13,-14,-12,-17,-18,38,-8,-9,-3,38,-6,38,-25,38,-26,38,-20,38,38,-10,-21,38,-27,38,-28,]),'RE':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,]),'SEC':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,]),'PI':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[12,12,12,12,12,12,12,12,12,12,12,12,12,12,12,12,]),'=':([1,5,12,17,19,21,22,23,24,25,26,27,28,29,31,32,33,40,41,42,43,44,45,46,47,48,49,50,52,54,55,56,57,59,60,62,63,],[-11,-4,-5,-7,39,-15,-19,-7,39,39,-2,-16,-13,-14,-12,-17,-18,39,-8,-9,-3,39,-6,-23,-25,-24,-26,-22,-20,39,39,-10,-21,39,-27,39,-28,]),'ACSC':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[13,13,13,13,13,13,13,13,13,13,13,13,13,13,13,13,]),'$end':([1,5,11,12,17,19,21,22,26,27,28,29,31,32,33,41,42,43,45,46,47,48,49,50,52,56,57,60,63,],[-11,-4,0,-5,-7,-1,-15,-19,-2,-16,-13,-14,-12,-17,-18,-8,-9,-3,-6,-23,-25,-24,-26,-22,-20,-10,-21,-27,-28,]),'FUNCTION':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[14,14,14,14,14,14,14,14,14,14,14,14,14,14,14,14,]),'CSC':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,]),'IM':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[16,16,16,16,16,16,16,16,16,16,16,16,16,16,16,16,]),'VARIABLE':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[17,23,17,17,17,17,17,17,17,17,17,17,17,17,17,17,]),"'":([0,1,5,6,7,8,12,17,20,21,22,23,24,26,27,28,29,30,31,32,33,34,35,36,37,38,39,41,42,43,45,46,47,48,49,50,51,52,53,56,57,58,60,61,63,],[6,-11,-4,6,6,6,-5,-7,6,-15,-19,41,42,-2,-16,-13,-14,6,-12,-17,-18,6,6,6,6,6,6,-8,-9,-3,-6,-23,-25,-24,-26,-22,6,-20,6,-10,-21,6,-27,6,-28,]),'SQRT':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[18,18,18,18,18,18,18,18,18,18,18,18,18,18,18,18,]),}

_lr_action = { }
for _k, _v in _lr_action_items.items():
   for _x,_y in zip(_v[0],_v[1]):
      if not _x in _lr_action:  _lr_action[_x] = { }
      _lr_action[_x][_k] = _y
del _lr_action_items

_lr_goto_items = {'group':([0,3,4,6,7,8,9,10,13,15,16,18,20,30,34,35,36,37,38,39,51,53,58,61,],[5,21,22,5,5,5,27,28,29,31,32,33,5,5,5,5,5,5,5,5,5,5,5,5,]),'expression':([0,6,7,8,20,30,34,35,36,37,38,39,51,53,58,61,],[19,24,25,26,40,44,45,46,47,48,49,50,54,55,59,62,]),'statement':([0,],[11,]),}

_lr_goto = { }
for _k, _v in _lr_goto_items.items():
   for _x,_y in zip(_v[0],_v[1]):
       if not _x in _lr_goto: _lr_goto[_x] = { }
       _lr_goto[_x][_k] = _y
del _lr_goto_items
_lr_productions = [
  ("S' -> statement","S'",1,None,None,None),
  ('statement -> expression','statement',1,'p_statement_expr','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',150),
  ('expression -> - expression','expression',2,'p_expression_uminus','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',162),
  ('group -> ( expression )','group',3,'p_group_parentheses','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',166),
  ('expression -> group','expression',1,'p_expression_group','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',170),
  ('expression -> PI','expression',1,'p_expression_pi','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',205),
  ('expression -> expression POWER expression','expression',3,'p_expression_power','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',209),
  ('expression -> VARIABLE','expression',1,'p_expression_variable','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',218),
  ("expression -> ' VARIABLE '",'expression',3,'p_expression_variable2','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',222),
  ("expression -> ' expression '",'expression',3,'p_expression_expression','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',226),
  ('expression -> COMPLEX ( expression , expression )','expression',6,'p_expression_complex','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',230),
  ('expression -> NUMBER','expression',1,'p_expression_number','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',234),
  ('expression -> CSC group','expression',2,'p_expression_func','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',240),
  ('expression -> SEC group','expression',2,'p_expression_func','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',241),
  ('expression -> ACSC group','expression',2,'p_expression_func','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',242),
  ('expression -> ASEC group','expression',2,'p_expression_func','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',243),
  ('expression -> RE group','expression',2,'p_expression_func','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',244),
  ('expression -> IM group','expression',2,'p_expression_func','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',245),
  ('expression -> SQRT group','expression',2,'p_expression_func','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',246),
  ('expression -> CONJ group','expression',2,'p_expression_func','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',247),
  ('expression -> FUNCTION ( expression )','expression',4,'p_expression_function1','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',253),
  ('expression -> FUNCTION ( expression , expression )','expression',6,'p_expression_function2','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',268),
  ('expression -> expression = expression','expression',3,'p_expression_binop','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',283),
  ('expression -> expression + expression','expression',3,'p_expression_binop','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',284),
  ('expression -> expression - expression','expression',3,'p_expression_binop','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',285),
  ('expression -> expression * expression','expression',3,'p_expression_binop','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',286),
  ('expression -> expression / expression','expression',3,'p_expression_binop','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',287),
  ('expression -> FUNCTION ( expression , expression , expression )','expression',8,'p_expression_function3','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',295),
  ('expression -> FUNCTION ( expression , expression , expression , expression )','expression',10,'p_expression_function4','/grid_mnt/opt__sbg__data__data1/cms/mbuttign/THESE/Monotop/Prod/Prod_AOD/Prod_LHE/MadGraph5_v1_5_3/aloha/aloha_parsers.py',308),
]
